launchpad-services/launcher-documentation.yaml

```diff
@@ -1,8 +1,8 @@
 services:
 - hash: dd46582683577a509a8be339dd59f5b754ff2171
-  name: appdev-documentation
+  name: launcher-documentation
   parameters:
-    IMAGE: registry.devshift.net/openshiftio/appdev-documentation
+    IMAGE: registry.devshift.net/openshiftio/launcher-documentation
```

Comment:
The image path should be fabric8/launcher-documentation

Reply:
... after https://github.com/fabric8-launcher/launcher-documentation/pull/961 is merged. Before that, the image name is the same

Reply:
Updating.